modules/widgets.php

```diff
@@ -31,6 +31,8 @@ function jetpack_load_widgets() {
 	foreach( $widgets_include as $include ) {
 		include $include;
 	}
+
+	include_once dirname( __FILE__ ) . '/widgets/migrate-to-core/image-widget.php';
```

Comment:
Should we maybe run the `is_admin()` and option check conditional here, before including the extra file?